Library/Formula/uim.rb

```diff
@@ -1,16 +1,20 @@
-require 'formula'
-
 class Uim < Formula
-  homepage 'http://code.google.com/p/uim/'
-  url 'https://uim.googlecode.com/files/uim-1.6.0.tar.bz2'
-  sha1 'd27f2ca8136da0702c82f0522911d06b2b8f8ea7'
+  homepage "https://github.com/uim/uim"
+  url "https://uim.googlecode.com/files/uim-1.6.0.tar.bz2"
+  sha256 "2a34dca2091eb6d61f05dabd8512c6658d8cefa8db14b7a684fbb10caea4a3aa"
+
+  head do
+    url "https://github.com/uim/uim.git"
+    depends_on "intltool"
+  end
 
-  depends_on 'pkg-config' => :build
-  depends_on 'gettext'
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
 
   def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!